dot/core/messages_test.go

```diff
@@ -75,40 +74,81 @@ func TestService_ProcessBlockAnnounceMessage(t *testing.T) {
 	require.Equal(t, expected, net.Message)
 }
 
-func TestService_HandleTransactionMessage(t *testing.T) {
-	// this currently fails due to not being able to call validate_transaction
+func createExtrinsics(t *testing.T, rt runtime.Instance, genHash common.Hash, nonce uint64) types.Extrinsic {
+	t.Helper()
+	bob, err := ctypes.NewAddressFromHexAccountID("0x90b5ab205c6974c9ea841be688864633dc9ca8a357843eeacf2314649965fe22")
```

Comment:
this is fine, but you can also get the key with `keystore.NewSr25519Keyring().Bob().Public().Hex()`

Reply:
Done

Reply:
Done